chart/gatekeeper-operator/values.yaml

```diff
@@ -0,0 +1,9 @@
+replicas: 1
+auditInterval: 60
+constraintViolationsLimit: 20
+image:
+  repository: quay.io/open-policy-agent/gatekeeper
+  release: v3.0.4-beta.2
```

Comment:
let's update this to the latest release so it will work with the gatekeeper.yaml in the chart